components/automate-cli/pkg/verifyserver/handlers/status/status.go

```diff
@@ -0,0 +1,40 @@
+package status
+
+import (
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/logger"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+type Status struct {
+	Logger logger.ILogger
+}
+
+type IStatus interface {
+	GetStatus(c *fiber.Ctx)
+}
+
+type StatusServices struct {
+	ServiceName string `json:"service_name"`
+	Status      string `json:"status"`
+	Version     string `json:"version"`
+}
+
+type StatusResponse struct {
+	Status   string           `json:"status"`
+	Services []StatusServices `json:"services"`
+}
+
+func NewHandler(logger logger.ILogger) IStatus {
```

Comment:
No need of this method, we can create a instance for status in router and then call the GetStatus handler.